Marlin/Conditionals_post.h

```diff
@@ -731,7 +731,7 @@
    * Set granular options based on the specific type of leveling
    */
 
-  #define UBL_DELTA  (ENABLED(AUTO_BED_LEVELING_UBL) && ENABLED(DELTA))
+  #define UBL_DELTA  (ENABLED(AUTO_BED_LEVELING_UBL) && (ENABLED(DELTA) || ENABLED(UBL_DELTA_ON_CARTESIAN)))
```

Comment:
There are now two different UBL line routines that work on cartesian.  Roxy's original version, which segments on mesh cell boundaries.  And the new "delta" routine that segments on granular boundaries (0.1mm default on delta, 1.0mm default on cartesian).  For a cartesian machine, the "delta" UBL line routine will "hug" the mesh within the cell instead of jumping from one cell wall intercept height to another, but probably at the cost of lower maximum feedrate.  The "current" UBL cartesian routine, Roxy's routine, should be the default for cartesian.  But there needs to be some flag to switch to the "delta" routine or "granular" routine or "segmented" routine used by deltas so cartesian can TRY it.  The name I chose is not the permanent name, and not advertised in config.h.  I just need something to let people try it on cartesians.